pkgs/by-name/ze/zeroad-unwrapped/package.nix

```diff
@@ -4,7 +4,7 @@
   perl,
   fetchurl,
   python3,
-  fmt,
+  fmt_9,
```

Comment:
https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults

such overrides is preferred to keep in all-packages.nix

Reply:
Switched back to overriding in top-level